packages/email-components/src/scss/tokens/_typography.scss

```diff
@@ -27,4 +27,4 @@ $font-weight-normal: 400;
 $font-weight-bold: 700;
 
 // Font families
-$font-family-base: arial, sans-serif;
+$font-family-base: arial, helvetica, sans-serif;
```

Comment:
This change is something new? I thought it'll be OK to use values from DS tokens - we agreed on this last week - Arial and sans-serif should do just fine.

I see a lot of inconsistencies - see screenshot (not only the highlighted line).

![Screenshot 2023-05-22 at 14 34 30](https://github.com/lmc-eu/jobs-design-system/assets/9672439/6b29a632-f4c2-48bb-b435-f0dde3d2e94a)

Reply:
Thanks for this check, now it is consistent.